sections/semantics-document-metadata.include

```diff
@@ -490,7 +496,9 @@
       <var>url</var> and <var>corsAttributeState</var>.
   5. Set <var>request</var>'s <a>client</a> to the <{link}> element's <a>node document</a>'s
       {{Window}} object's <a>environment settings object</a>.
-  6. <a>Fetch</a> <var>request</var>.
+  6. Set <var>request</var>'s <span>cryptographic
+      nonce metadata</span> to the current state of the <{link}> element's <a>nonce</a> content attribute.	  
```

Comment:
You'll need to declare what kind of `nonce` you're looking for. Probably something like `<a element-attr for="link">`, or the shortcut `<{link/nonce}>` (which will produce prettier output).